.github/workflows/cd-deploy-www-staging.yml

```
@@ -3,10 +3,17 @@ on:
   push:
     branches:
       - master
+    paths:
+      - 'sites/website/**'
+      - '.github/workflows/cd-deploy-www-staging.yml'
   pull_request:
     types: [opened, synchronize, reopened, closed]
     branches:
       - master
+    paths:
+      - 'sites/website/**'
```

Should we include the frontpage?
```suggestion
      - 'sites/fast-website/**'
      - 'sites/website/**'
```